src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -522,8 +522,8 @@ export const GuiPluginListScene = React.memo((props: OwnProps) => {
     dispatch(logEvent(event, values))
   })
 
-  const handleCountryPress = useHandler(() => {
-    dispatch(
+  const handleCountryPress = useHandler(async () => {
+    await dispatch(
```

Comment:
This works at run-time, but there seems to be a type mis-match. The return type of `showCountrySelectionModal` should be `ThunkAction<Promise<void>>` instead of `ThunkAction<void>`. If you fix that, then a few more red squiggles appear at other places that have this same debounce issue.